wordpress-mobile/WordPress-Android
# Add tracking for stats inconsistency

Helps with #11412 

With the previous attempts I've added logs to find out what's happening when the user gets inconsistent stats. I'm now suspicious of some API caching so I'm adding tracking for the cases when the app is showing old data to the user. The threshold is 2 days so let's see if we get any relevant information out of this. I've added a field for checking the age of visible stats so we can filter out users who haven't been online in a while. 

To test:
- Go to Stats/Days tab
- Check that the overview (first) block is showing you the correct dates and nothing is crashing there
- The tracked error should not be visible in the logs - filter out `stats_overview_error`

PR submission checklist:
